applications/openshift/networking/var_network_policies_namespaces_whitelist_regex.var

```diff
@@ -0,0 +1,18 @@
+documentation_complete: true
+
+title: 'Whitelist Namespaces for Network Policies'
```

Comment:
Let's use more inclusive and descriptive wording:

```suggestion
title: 'Namespaces Allowed Without Network Policy'
```

or
```suggestion
title: 'Namespaces exempt of Network Policies'
```

Don't  forget to rename the variable file accordingly.

Reply:
@Vincent056 You forgot to rename the variable and the references to it, :grin: